File: models/order.ts
import { IPurchase } from "./purchase";

export interface IOrder {
  // seller:string;  // sprzedajacy
  order: string; //zamowienie
  isAllegroPay: string;// allegro Pay	
  isBusinessOrder: string; //na firme?
  orderTotalAmount: string;// suma kupionych

  isInvoiceReceived: string;// faktura OK ?
  isPackageReceived: string; // Przesyłka odebrana
  receivedDate?: string;// Data odebrania -> 

  isReturnIssued: string; //Zwrot zgłoszony

  returnToDate: string;// odeslij do
  returnCode: string;// kod zwrotu
  returnTotalAmount: string; // kwota do zwrotu

  isReturnSended: string;// Zwrot wysłany
  isReturnDelivered: string;// Zwrot u sprzedającego
  isCashReturned: string;// zwrot na koncie
  isInvoiceCorrectionReceived: string;// korekta OK ?

  isFinished: string; // ALL OK
  note: string; // notatka
}

File: OgarniaczAllegro-Backend/OgarniaczAllegro.DAL/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OgarniaczAllegro.DAL.Models
{
    internal class Order
    {
        // seller:string;  // sprzedajacy
        string Name { get; set; } //zamowienie
        string IsAllegroPay { get; set; } // allegro Pay	
        string IsBusinessOrder { get; set; } //na firme?
        string OrderTotalAmount { get; set; } // suma kupionych

        string IsInvoiceReceived { get; set; } // faktura OK ?
        string IsPackageReceived { get; set; } // Przesyłka odebrana
        string ReceivedDate { get; set; } // Data odebrania -> 

        string IsReturnIssued { get; set; } //Zwrot zgłoszony

        string ReturnToDate { get; set; } // odeslij do
        string ReturnCode { get; set; } // kod zwrotu
        string ReturnTotalAmount { get; set; } // kwota do zwrotu

        string IsReturnSended { get; set; } // Zwrot wysłany
        string IsReturnDelivered { get; set; } // Zwrot u sprzedającego
        string IsCashReturned { get; set; } // zwrot na koncie
        string IsInvoiceCorrectionReceived { get; set; } // korekta OK ?

        string IsFinished { get; set; } // ALL OK
        string Note { get; set; } // notatka
    }
}
